lib/mavenlink/client.rb

```diff
@@ -81,7 +81,8 @@ def connection_options
                    "User-Agent" => user_agent.to_s,
                    "Authorization" => "Bearer #{oauth_token}" },
         ssl: { verify: false },
-        url: endpoint
+        url: endpoint,
+        request: { timeout: 180 }
```

Comment:
Can we make this configurable? It might fit in something like `Mavenlink::Settings::DEFAULT`.

Reply:
shouldn't need to make configurable.

Reply:
Okay I'm cool either way. I think it would still be beneficial to at least make the value a constant with a descriptive name. Maybe something like `MAVENLINK_API_TIMEOUT = 180`.